Project/frontend/Dockerfile.frontend

```diff
@@ -0,0 +1,15 @@
+FROM node:16-alpine 
+
+WORKDIR /app
+
+ENV PATH /app/node_modules/.bin:$PATH
+
+COPY package.json ./
+COPY package-lock.json ./
+
+RUN npm install --silent
+RUN npm install
```

Comment:
Czemu jest 2 razy npm install? Czy to jest jakiś safety lock?

Reply:
faktycznie nie ma to sensu - albo copilot źle podpowiedział albo skopiowałem nie to co trzeba